app/assets/javascripts/modules/_admin.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Admin module
+ *
+ */
+var admin = (function() {
+
+  return {
+    
+    handleTableLinks: function() {
```

Comment:
Wrapping a `tr` with an `a` tag is not valid HTML and the DOM will reconstruct itself accordingly. Another solution besides using javascript would be using `link_to` within `td` elements, which I actually prefer. Regarding the JS breaking, I'm still sorting out the best ways to structure and initialize functions to work consistently when linking the javascript the page `head` and using turbolinks